puppet/manifests/default.pp

```diff
@@ -30,8 +79,27 @@
   }
 }
 
+# This needs to be on exactly two nodes.
+class gluster_cluster {
+  include glusterfs_common
+
+  gluster_peer { ['controller.seed-stack.local', 'worker.seed-stack.local']: }
```

Comment:
Cool. Can you show me where the autorequire is defined in the puppet-gluster code, please?